templates/miq-template.yaml

```diff
@@ -149,29 +150,14 @@ objects:
               name: "DATABASE_REGION"
               value: "${DATABASE_REGION}"
             -
-              name: "DATABASE_NAME"
```

Comment:
We need all of the database parameters for the PG pod, they are used to create the database, user and password along with tunables, check on template, PG pod objects.